### ISSUE TITLE
How do I run Vitebook from the root of an existing project

### ISSUE THREAD

**Issue by Soya-xy** [vitebook/vitebook#17]
When I run `yarn create vitebook` in an existing project, it doesn't work
![image](https://user-images.githubusercontent.com/56671014/149734558-785cb8b0-9d02-4f4a-86fa-c2106fd786c7.png)

**Comment by mihar-22** [comment]
Can you give me a repro please? The project you ran the command in so I can see why it failed.

**Comment by Soya-xy** [comment]
Is this report?
```
❯ yarn vitebook:dev
yarn run v1.22.17
warning package.json: No license field
$ vitebook dev
下午6:01:25 [vite-plugin-svelte] invalid plugin options "useVitePreprocess" in config
{
  extensions: [ '.svelte', '.md' ],
  useVitePreprocess: true,
  preprocess: [
    { markup: [AsyncFunction: markup] },
    { markup: undefined },
    { markup: undefined },
    { markup: undefined }
  ],
  compilerOptions: { hydratable: false }
}
vitebook@0.19.3

Port 3000 is in use, trying another one...
  > Local: http://localhost:3001/
  > Network: use `--host` to expose

下午6:01:27 [vite] Internal server error: /Users/sys/Desktop/work/code/ldui2/node_modules/@vitebook/client/dist/client/App.svelte:28:16 Cannot have an {:else} block outside an {#if ...} or {#each ...} block
  Plugin: vite-plugin-svelte
  File: /Users/sys/Desktop/work/code/ldui2/node_modules/@vitebook/client/dist/client/App.svelte
   26 |  const file = "node_modules/@vitebook/client/dist/client/App.svelte";
   27 |  
   28 |  // (17:0) {:else}
                         ^
   29 |  function create_else_block(ctx) {
   30 |    let switch_instance;
下午6:01:27 [vite] Internal server error: /Users/sys/Desktop/work/code/ldui2/node_modules/@vitebook/client/dist/client/components/DefaultNotFound.svelte:18:1 The keyword 'let' is reserved
  Plugin: vite-plugin-svelte
  File: /Users/sys/Desktop/work/code/ldui2/node_modules/@vitebook/client/dist/client/components/DefaultNotFound.svelte
   16 |  
   17 |  function create_fragment(ctx) {
   18 |    let div;
           ^
   19 |  
   20 |    const block = {

```
After using vite to create a new svelte template, I created a vitebook under the current project using yarn create vitebook. I execute yarn vitebook: dev to prompt this error

**Comment by mihar-22** [comment]
Thanks, if I have time after some work I'm doing tonight I'll have a look, otherwise best left until tomorrow.

**Comment by Soya-xy** [comment]
thank You Very Much

**Comment by mihar-22** [comment]
I had a quick look at this, it's because the Svelte Vite plugin is passed twice. Vitebook runs Vite under the hood which still picks up on the root config `vite.config.ts`, and then it ends up loading the plugin twice, once by Vitebook `@vitebook/client`, and the other inside `vite.config.ts`. 

Not entirely sure yet how to approach handling this.

**Comment by mihar-22** [comment]
I think this should be now resolved on the latest `>=0.19.5`. If you can test it out and let me know that'd be awesome! CLosing for now, happy to reopen.
